week 5/README.md

```diff
@@ -190,3 +190,25 @@ Output
 1 2 3
 ```
 
+
+### Task 3 - WTF Cipher
```

Comment:
1. I didn't wanted to name Vigenere's cipher explicitly because it will be easier for them to find the solution.
2. Yes I also wanted to add decoding part, but was not sure how to formulate the task.